x-pack/plugins/canvas/public/lib/aeroelastic/dag_start.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { select } from './select';
+
+const state = d => d; // serves as reminder that we start with the state; todo remove it as we add TS annotations (State)
```

Comment:
nit: please put `TODO:` comments on their own line, it makes them easier to find later